src/compo/viirs_l1bnc2ioda.py

```diff
@@ -170,7 +181,8 @@ def _read(self):
                 err = obsgrp.variables[errname]
                 err.set_auto_scale(False)
 
-                vals[:, ichan] = obs[:].data.ravel()
+                # NASA VIIRS apply secant if option enabled
+                vals[:, ichan] = obs[:].data.ravel() * sec_term if self.apply_secterm else obs[:].data.ravel()
```

Comment:
@weiwilliam separating the logic will help for clarity,, but now whoops need to add the location check for @gthompsnJCSDA